README.md

```diff
@@ -69,6 +69,8 @@ There's also a test for checking whether log size is below or above threshold.
 
 ## How you can use it?
 
+Please install [Python 2.7](https://www.python.org/downloads/) and [Ruby](https://www.ruby-lang.org/en/documentation/installation/).
```

Comment:
NS is also working with Python 3+ but `pypy` is required. Can you please change it accordingly?